examples/lsodar_bouncing_ball.py

```diff
@@ -15,12 +15,10 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-import numpy as N
 import nose
+import numpy as N
```

Comment:
I assume we will change this `as N` to `as np` in another ticket, feel free to resolve.

Reply:
Yes, on my To-do list.